docs/configuration/recording_rules.md

```diff
@@ -45,8 +45,10 @@ dashboards, which need to query the same expression repeatedly every time they
 refresh.
 
 Recording and alerting rules exist in a rule group. Rules within a group are
-run sequentially at a regular interval. The names of recording rules
-must be [valid metric names](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
+run sequentially at a regular interval, over the same evaluation time.
+That way, the results of rules within a group are consistent even if one rule is slow.
```

Comment:
I'd remove this, the design reasoning here is more complicated than that and this is not the place for it.